plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/api/color/ColorManager.java

```diff
@@ -113,48 +111,6 @@ private String getKey(RGBValues color) {
         return sb.toString();
     }
 
-    /**
-     * Collect the {@link UserFixedColor}s defined in the all the selected VSM of the given session and concat them to
-     * the {@link SystemColor}s.
-     * 
-     * @param session
-     *            the session in which to find UserFixedColor
-     * @return A map which key is the color name and the value is the RGB.
-     */
-    public Map<String, RGB> collectVsmAndDefaultColors(Session session) {
```

Comment:
This method is in the API, please update the release note.

Reply:
done